mod-gobi/schemas/mapping.json

```diff
@@ -588,8 +583,13 @@
               "description": "Lookup acquisition method UUIDs"
             },
             {
-              "name": "LOOKUP_ACQUISITION_UNIT_DEFAULT_ACQ_UNIT_NAME",
-              "title": "lookupAcquisitionUnitDefaultAcqUnitName",
+              "name": "LOOKUP_ACQUISITION_UNIT_DEFAULT",
+              "title": "lookupAcquisitionUnitDefault",
```

Comment:
I think this translator should be renamed also, basically we should allow user to find acquisition unit UUID by any acquisition unit name, so DEFAULT word should be replaced

Reply:
Also to be consistent with other translators we need to add ID into naming for both new translators

Reply:
yes, make sense. 
fixed .